src/Framework/Gacela.php

```diff
@@ -4,18 +4,22 @@
 
 namespace Gacela\Framework;
 
+use Gacela\Framework\Config\ConfigReaderInterface;
+
 final class Gacela
 {
     /**
      * Define the entry point of Gacela.
-     * Mainly to define the application root directory and optional global services.
      *
      * @param array<string,mixed> $globalServices
+     * @param array<string,ConfigReaderInterface> $configReaders
      */
-    public static function bootstrap(string $appRootDir, array $globalServices = []): void
+    public static function bootstrap(string $appRootDir, array $globalServices = [], array $configReaders = []): void
     {
         Config::getInstance()
+            ->setAppRootDir($appRootDir)
             ->setGlobalConfigServices($globalServices)
-            ->init($appRootDir);
+            ->setConfigReaders($configReaders)
+            ->init();
```

Comment:
I really didn't like anymore the `Config::setConfigReaders()`. Now it looks awesome 👏